main/lib/src/Event.cc

```diff
@@ -14,7 +14,11 @@ namespace eudaq {
       "EORE",
       "HITS",
       "FAKE",
-      "SIMU"
+      "SIMU",
+      "EUDAQ2",
+      "PACKET",
```

Comment:
@RPeschke shouldn't the `FLAG_PACKET` be gone by now? Or did you only remove it from 2.0-dev?